models/iaf_psc_alpha.h

```diff
@@ -40,71 +40,98 @@ namespace nest
 +++++++++++++++++
 
-Leaky integrate-and-fire neuron model
+Leaky integrate-and-fire model with alpha-shaped input currents
 
 Description
 +++++++++++
 
-``iaf_psc_alpha`` is an implementation of a leaky integrate-and-fire model
-with alpha-function shaped synaptic currents. Thus, synaptic currents
-and the resulting postsynaptic potentials have a finite rise time.
+``iaf_psc_alpha`` is a leaky integrate-and-fire neuron model with
 
-The threshold crossing is followed by an absolute refractory period
-during which the membrane potential is clamped to the resting potential.
+* a hard threshold,
+* a fixed refractory period,
+* no adaptation mechanisms,
+* :math:`\alpha`-shaped synaptic input currents.
 
-The linear subthreshold dynamics is integrated by the Exact
-Integration scheme [1]_. The neuron dynamics is solved on the time
-grid given by the computation step size. Incoming as well as emitted
-spikes are forced to that grid.
+The membrane potential evolves according to
 
-An additional state variable and the corresponding differential
-equation represents a piecewise constant external current.
+.. math::
 
-The general framework for the consistent formulation of systems with
-neuron like dynamics interacting by point events is described in
-[1]_.  A flow chart can be found in [2]_.
+   \frac{dV_m}{dt} = -\frac{V_m - E_L}{\tau_m} + \frac{I_{\text{syn}} + I_e}{C_m}
 
-Critical tests for the formulation of the neuron model are the
-comparisons of simulation results for different computation step
-sizes and the testsuite contains a number of such tests.
+where the synaptic input current :math:`I_{\text{syn}}(t)` is discussed below and :math:`I_e` is
+a constant input current set as a model parameter.
 
-The ``iaf_psc_alpha`` is the standard model used to check the consistency
-of the nest simulation kernel because it is at the same time complex
-enough to exhibit non-trivial dynamics and simple enough compute
-relevant measures analytically.
+A spike is emitted at time step :math:`t^*=t_{k+1}` if
 
-.. note::
+.. math::
+
+   V_m(t_k) < V_{th} \quad\text{and}\quad V_m(t_{k+1})\geq V_{th} \;.
+
+Subsequently,
+
+.. math::
+
+   V_m(t) = V_{\text{reset}} \quad\text{for}\quad t^* \leq t < t^* + t_{\text{ref}} \;,
+
+i.e., the membrane potential is clamped to :math:`V_{\text{reset}}` during the refractory period.
+
+The synaptic input current has an excitatory and an inhibitory component
+
+.. math::
+
+   I_{\text{syn}}(t) = I_{\text{syn, ex}}(t) + I_{\text{syn, in}}(t)
+
+where
+
+.. math::
 
-   The present implementation uses individual variables for the
-   components of the state vector and the non-zero matrix elements of
-   the propagator. Because the propagator is a lower triangular matrix,
-   no full matrix multiplication needs to be carried out and the
-   computation can be done "in place", i.e. no temporary state vector
-   object is required.
+   I_{\text{syn, ex}}(t) = \sum_{(t_m, w_m)\\w_m\geq 0} w_m i_{\text{syn, ex}}(t-t_m)
```

Comment:
I don't like m as a summation index here, since we're already using it as a subscript for membrane parameters. I also think the summation notation is a bit confusing.

```suggestion
.. math::

   I_{\text{syn, ex}}(t) = \sum_{j} w_j \sum_k i_{\text{syn, ex}}(t-t_j^k-d_j) \;,

where :math:`j` indexes excitatory presynaptic neurons, :math:`k` indexes the spike times of neuron :math:`j`, and :math:`d_j` is the delay from neuron :math:`j`.

```

```suggestion
.. math::

   I_{\text{syn, ex}}(t) = \sum_{j \in \Gamma} w_j \sum_{t' \in \Pi_j} i_{\text{syn, ex}}(t-t'-d_j) \;,

where :math:`\Gamma` is the set of indices of excitatory presynaptic neurons, :math:`\Pi_j` is the set of spike times from neuron :math:`j`, and :math:`d_j` is the delay from neuron :math;`j`.

```

Reply:
I included you first suggestion.